### DISCUSSION TITLE
Can't import Page or Text from flet, any one can help me?

### DISCUSSION THREAD

**Discussion by HocfaiSun** [flet-dev/flet#416] [Q&A]
I have installed flet, when I copy the example code to run, it shows the error like this 'cannot import name 'Page' from partially initialized module 'flet' (most likely due to a circular import)', I try to solve this problem but failed, any one can help me with that? Thank you.

**Comment by FeodorFitsner** [comment]
What Python version do you run?
Could you please provide a code sample to reproduce the issue?

**Reply by HocfaiSun** [reply]
My python version is 3.9.13. 
I used this example code:

_**import flet
from flet import Page

def main(page: Page):
    # add/update controls on Page
    pass

flet.app(target=main)**_

However, when I run this simple code it doesn't work, it says 'ImportError: cannot import name 'Page' from partially initialized module 'flet' (most likely due to a circular import)'.

**Reply by FeodorFitsner** [reply]
What OS is that and the version of Flet?

**Reply by HocfaiSun** [reply]
The OS is windows and the version of Flet is 0.1.60.

**Reply by HocfaiSun** [reply]
I have tried to install Flet on different windows computers, however, each time it shows the same error. I tried just to import flet, it's ok, but when I import Page from flet, the error occurs.

**Reply by FeodorFitsner** [reply]
What command do you use to install it?

**Reply by HocfaiSun** [reply]
I used 'pip install flet'.

**Reply by HocfaiSun** [reply]
I tried the sample code in jupyter notebook, it works.

**Reply by HocfaiSun** [reply]
I use anaconda prompt to install this package, I run the sample code in jupyter notebook, it works well. However when I run the same code in vscode or spyder or pycharm whose environment is the same as jupyter notebook, all of them do not work. I don't know why this problem occurs, could you please give me some advice? Thank you.
